platform-tests/src/test/java/org/junit/platform/engine/support/hierarchical/ParallelExecutionIntegrationTests.java

```diff
@@ -283,6 +280,14 @@ void runsIsolatedTestsLastToMaximizeParallelism() {
 		assertThat(isolatedClassStart).isAfterOrEqualTo(parallelClassFinish);
 	}
 
+	@ParameterizedTest
+	@ValueSource(classes = { IsolatedMethodFirstTestCase.class, IsolatedMethodLastTestCase.class })
+	void canRunTestsIsolatedFromEachOtherWhenDeclaredOnFirstMethod(Class<?> testClass) {
```

Comment:
The `WhenDeclaredOnFirstMethod` seems to contradict the `IsolatedMethodFirstTestCase` and `IsolatedMethodLastTestCase`.

Reply:
Thanks for spotting that! I refactored from two test methods to a parameterized one and forgot to rename the method.

Reply:
```suggestion
	void canRunTestsIsolatedFromEachOtherWhenDeclaredOnMethodLevel(Class<?> testClass) {
```